lib/embed/embed.dart

```diff
@@ -674,7 +674,7 @@ class PlaygroundMobile {
     _output.element.innerHtml =
         "<div class='consoleTitle default-text-div layout horizontal center-center'>"
         "<span class='default-text'>Console output</span>"
```

Comment:
Missed a few here?

Reply:
Actually, how is the linter not triggering on these lines?

Reply:
The linter won't trigger because it'd need to escape the single quotes for the html attributes. We can flip these around, and I intend to, but in a follow up PR.